visualization/app/codeCharta/ui/edgeSettingsPanel/edgeMetricToggle/edgeMetricToggle.component.scss

```diff
@@ -0,0 +1,30 @@
+cc-edge-metric-toggle {
+	.mat-checkbox {
+		margin: 15px 0;
+	}
+
+	.mat-checkbox-layout {
+		padding-left: 4px;
+
+		.mat-checkbox-inner-container {
+			margin-right: 15px;
+		}
+	}
+
+	.mat-checkbox-indeterminate.mat-accent .mat-checkbox-background,
+	.mat-checkbox-checked.mat-accent .mat-checkbox-background,
+	.mat-checkbox-checked:not(.mat-checkbox-disabled).mat-accent .mat-ripple-element,
+	.mat-checkbox:active:not(.mat-checkbox-disabled).mat-accent .mat-ripple-element {
+		background-color: rgb(0, 150, 136);
+	}
+
+	.mat-checkbox-frame,
+	.mat-checkbox-background,
+	.mat-checkbox-anim-unchecked-checked .mat-checkbox-anim-checked-unchecked {
+		transform: scale(1.1);
+	}
+
+	mat-checkbox.edgeMetricCheckbox .mat-checkbox-checkmark-path {
+		stroke: black !important;
```

Comment:
important is evil, but if its to counteract weird material behaviour, its probably fine

Reply:
I'm so sorry, I found no other solution. For now we leave it like that.